sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderFactories.java

```diff
@@ -34,18 +32,14 @@
   private CoderFactories() { } // Static utility class
 
   /**
-   * Creates a {@link CoderFactory} built from particular static methods of a class that
-   * implements {@link Coder}.
-   *
-   * <p>The class must have the following static methods:
+   * Creates a {@link CoderFactory} for a given raw type which is able to build a coder from a
```

Comment:
The term "raw type" refers to the relationship between an actual type and a class. The phrasing is also ambiguous - the `CoderFactory` should work for more than raw types, that's the whole point.

Since it is now just one static method, how about just shortening it to:

`"Creates a {@link CoderFactory} from a class's {@code static <T> Coder<T> of(TypeDescriptor<T>, List<Coder<?>>}) method."`

You probably want to switch from varargs to list here. That will then allow a single factory to provide coders for types with various numbers of type args.